src/main/groovy/grails/plugins/airbrake/AirbrakeNotifier.groovy

```diff
@@ -11,14 +11,14 @@ class AirbrakeNotifier {
     def grailsApplication
 
     static final String AIRBRAKE_API_VERSION = '2.2'
-	static final String AIRBRAKE_HOST = 'airbrakeapp.com'
-	static final String AIRBRAKE_PATH = '/notifier_api/v2/notices'
+    static final String AIRBRAKE_HOST = 'airbrakeapp.com'
+    static final String AIRBRAKE_PATH = '/notifier_api/v2/notices'
 
-	static final String NOTIFIER_NAME = 'grails-airbrake'
-	static final String NOTIFIER_VERSION = '1.0'
-	static final String NOTIFIER_URL = 'https://github.com/cavneb/airbrake-grails'
+    static final String NOTIFIER_NAME = 'grails-airbrake'
+    static final String NOTIFIER_VERSION = '0.9.4'
+    static final String NOTIFIER_URL = 'https://github.com/cavneb/airbrake-grails'
 
-    final Configuration configuration
+    final grails.plugins.airbrake.Configuration configuration
```

Comment:
It doesn't need to be imported either as it's in the same package. Sometimes IntelliJ does this automatically. I have not noticed it before.